lib/qb_integration/services/line.rb

```diff
@@ -91,16 +91,16 @@ def build_from_line_items(account = nil)
             #   end
             # end
           else
-            unless line_item["price"] && line_item["quantity"]
+            unless line_item["line_item_price"] && line_item["quantity"]
               raise UnsupportedException.new "Line Items must have a valid price and quantity"
             end
-            line.amount = (line_item["quantity"].to_i * line_item["price"].to_f)
+            line.amount = (line_item["quantity"].to_i * line_item["line_item_price"].to_f)
             line.description = line_item["name"]
 
             line.sales_item! do |sales_item|
               sales_item.item_id = item_found.id
               sales_item.quantity = line_item["quantity"].to_i
-              sales_item.unit_price = line_item["price"].to_f
+              sales_item.unit_price = line_item["line_item_price"].to_f
```

Comment:
Maybe we should be backwards compatible here too?
`sales_item.unit_price = price(line_item).to_f`
```
def price
line_item["line_item_price"] || line_item["price"]
end```

Reply:
wdyt?

Reply:
Agreed! That would be a better way to do that and you're on a roll 😀